week3/homework/src/actions/deserializeTodo.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+async function deserializeTodo(request) {
+  const { todo } = request.body;
+  if (todo == null) throw new Error('todo not set');
+
+  if (todo.description != null) todo.description = todo.description.trim();
+
+  if (todo.description == null || todo.description.length === 0)
+    throw new Error('description not set');
```

Comment:
what would happen if a user tries to create a new todo with an empty description?
Is this the correct behaviour?